tasks/main.yml

```diff
@@ -0,0 +1,42 @@
+---
+
+- name: Install apt-transport-https
+  become: yes
+  apt: 
+    name: "{{ item }}"
+  with_items:
+    - apt-transport-https
+
+- name: Add Elasticsearch repository key
```

Comment:
why?

Reply:
Because everything from the elk stack is signed with the elasticsearch key